click_odoo_contrib/restoredb.py

```diff
@@ -35,6 +35,8 @@ def _restore_from_folder(dbname, backup, copy=True, jobs=1):
         # if it's a copy of a database, force generation of a new dbuuid
         reset_config_parameters(dbname)
     with OdooEnvironment(dbname) as env:
+        if neutralize and odoo.release.version_info >= (16, 0):
+            odoo.modules.neutralize.neutralize_database(env.cr)
```

Comment:
This should fail if the neutralize option is not supported otherwise this gives a false sense of success.

Reply:
You're right, I'll change it to fail in case it's not supported.

Reply:
Done, I've added the check for the neutralize option and the odoo version in the main function. I've also added the tests